oracle-linux-image-tools/distr/ol8-slim/image-scripts.sh

```diff
@@ -41,6 +42,11 @@ distr::validate() {
 distr::kickstart() {
   local ks_file="$1"
 
+  local btrfs="\
+part btrfs.01 --fstype=\"btrfs\"  --ondisk=sda --size=4096 --grow\n\
+btrfs none --label=btr_pool --data=single btrfs.01\n\
```

Comment:
`btr` is not a suitable shortened term and `pool` isn't a term used with `btrfs` either. If we can't use the default label of `ol_${hostname}`, I suggest using `btrfs_vol` as the terms `volume` and `subvolume` are used with btrfs.